tests/test_updater.py

```diff
@@ -2065,7 +2044,6 @@ def test_get_valid_targetinfo(self):
     shutil.rmtree(os.path.join(self.repository_directory2, 'metadata'))
     shutil.copytree(os.path.join(self.repository_directory2, 'metadata.staged'),
         os.path.join(self.repository_directory2, 'metadata'))
-
```

Comment:
Nit: I'm curious why this empty line was removed?

Reply:
Honestly, it seems it's a mistake I did while I was resolving prs.
I will fix it right away.